third_party/xxhash.BUILD

```diff
@@ -0,0 +1,10 @@
+cc_library(
+    name = "xxhash",
+    srcs = glob(["xxhash.c"]),
+    hdrs = glob(["xxhash.h"]),
+    linkstatic = select({
+        "@com_stripe_ruby_typer//tools/config:linkshared": 0,
+        "//conditions:default": 1,
+    }),
+    visibility = ["//visibility:public"],
+)
```

Comment:
Can you make sure that jump to definition works with these functions?

Reply:
We have many automated tests for jump-to-def. What's your concern?

Reply:
Sorry, I meant clangd jump to def. I think you'll need to update some checked in files when you add new bazel targets.

Reply:
It works 👍